db/seeds/utils.js

```diff
@@ -42,6 +42,16 @@ exports.createCommentNumber = (articles) => {
       return articles
 
       })
-  }
+}
+  
```

Comment:
If you're going to create util functions, there should be tests for util functions.